src/js/signin/containers/Signin.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+
+class Main extends React.Component {
```

Comment:
`class Signin`?

Reply:
Actually, maybe make this class `SignIn` (with the capitalized "In") for proper camel-casing.